tests/emails/beneficiary_warning_after_pro_booking_cancellation_test.py

```diff
@@ -0,0 +1,163 @@
+from datetime import datetime, timedelta, timezone
+from unittest.mock import patch
+
+from emails.beneficiary_warning_after_pro_booking_cancellation import \
+    retrieve_data_to_warn_beneficiary_after_pro_booking_cancellation
+from tests.test_utils import create_booking, create_stock_from_event_occurrence, create_event_occurrence, \
+    create_offer_with_event_product, create_venue, create_offerer, create_user, create_offer_with_thing_product, \
+    create_stock_from_offer
+
+
+class RetrieveDataToWarnBeneficiaryAfterProBookingCancellationTest:
+    @patch('emails.beneficiary_warning_after_pro_booking_cancellation.feature_send_mail_to_users_enabled', return_value=True)
+    def test_should_when_send_mail_to_user_when_feature_send_mail_to_users_is_enabled(self,
+                                                                                      feature_send_mail_to_users_enabled,
+                                                                                      app):
+        # Given
+        beginning_datetime = datetime(2019, 7, 20, 12, 0, 0, tzinfo=timezone.utc)
+        end_datetime = beginning_datetime + timedelta(hours=1)
+        booking_limit_datetime = beginning_datetime - timedelta(hours=1)
+
+        user = create_user(public_name='John Doe')
+        offerer = create_offerer(name='Test offerer')
+        venue = create_venue(offerer)
+        offer = create_offer_with_event_product(venue, event_name='Mains, sorts et papiers')
+        event_occurrence = create_event_occurrence(offer, beginning_datetime=beginning_datetime,
+                                                   end_datetime=end_datetime)
+        stock = create_stock_from_event_occurrence(event_occurrence, price=20, available=10,
+                                                   booking_limit_date=booking_limit_datetime)
+        booking = create_booking(user, stock)
+
+        # When
+        mailjet_data = retrieve_data_to_warn_beneficiary_after_pro_booking_cancellation(booking)
+
+        # Then
+        assert mailjet_data['To'] == user.email
```

Comment:
Voici ce que donne ce test en enlevant tous les bruits :
```suggestion
        # Given
        user = create_user()
        offerer = create_offerer()
        venue = create_venue(offerer)
        offer = create_offer_with_event_product(venue)
        event_occurrence = create_event_occurrence(offer)
        stock = create_stock_from_event_occurrence(event_occurrence)
        booking = create_booking(user, stock)

        # When
        mailjet_data = retrieve_data_to_warn_beneficiary_after_pro_booking_cancellation(booking)

        # Then
        assert mailjet_data['To'] == user.email
```
Il ne faut mettre que l'essentiel correspondant à ton test pour favoriser la lecture.
Peux tu faire une passe sur les autres tests en ce sens stp ?

Reply:
:+1: